### PR TITLE
KubernetesV1: Updating from vsts-task-lib@2.0.5 to azure-pipelines-ta…

### PR DESCRIPTION
**Pull request by savenkat**
…sk-lib@2.8.0

- Also moved KubernetesV1
	- from 'kubernetes-common' to 'kubernetes-common-v2'
	- from 'docker-common' to 'docker-common-v2'
	- from 'azure-arm-rest' to 'azure-arm-rest-v2'
	- from 'utility-common' to 'utility-common-v2'
- Bumped up patch version

### CODE REVIEW

**Comment by ds-ms** [comment]
Given that `kubernetes-common` is not being used by any other task, we can remove the folder altogether, right?

File: Tasks/KubernetesV1/make.json [hunk 1]
```diff
@@ -27,11 +27,11 @@
...
             "items": [
-                "node_modules/utility-common/node_modules/vsts-task-lib",
-				"node_modules/docker-common/node_modules/vsts-task-lib",
+                "node_modules/utility-common-v2/node_modules/azure-pipelines-task-lib",
+				"node_modules/docker-common-v2/node_modules/azure-pipelines-task-lib",
 				"node_modules/vsts-task-tool-lib/node_modules/vsts-task-lib",
-				"node_modules/azure-arm-rest/node_modules/vsts-task-lib",
-				"node_modules/kubernetes-common/node_modules/vsts-task-lib"
+				"node_modules/azure-arm-rest-v2/node_modules/azure-pipelines-task-lib",
+				"node_modules/kubernetes-common-v2/node_modules/azure-pipelines-task-lib"
```
Comment (ds-ms) [inline Tasks/KubernetesV1/make.json]: NIT: indent?

File: Tasks/KubernetesV1/npm-shrinkwrap.json [hunk 2]
```diff
@@ -529,46 +489,37 @@
       }
     },
     "vsts-task-lib": {
```
Comment (ds-ms) [inline Tasks/KubernetesV1/npm-shrinkwrap.json]: The task shouldn't have reference to `vsts-task-lib` right?

**Comment by savenkat** [comment]
> Given that `kubernetes-common` is not being used by any other task, we can remove the folder altogether, right?

kubernetesV0 and helmInstallerV0 are still using it

File: Tasks/KubernetesV1/npm-shrinkwrap.json (hunk 2 above)
Comment (savenkat) [reply Tasks/KubernetesV1/npm-shrinkwrap.json]: vsts-task-tool-lib has an internal reference to vsts-task-lib

File: Tasks/KubernetesV1/make.json (hunk 1 above)
Comment (savenkat) [reply Tasks/KubernetesV1/make.json]: will fix in a subsequent check-in. thnx.

**Pull request merged**